### DISCUSSION TITLE
[Feature Request] Check if PR description exists

### DISCUSSION THREAD

**Discussion by anthonyma94** [GsActions/commit-message-checker#65] [General]
Hi, I'm wondering if there can be a boolean flag to check if a PR description exists. I'd like to be able to check if a description exists given a specific pattern for PR title :)

**Comment by gilbertsoft** [comment]
@anthonyma94 can you please provide some more information, I don't fully understand what you are trying to achieve. Some examples may be good...

**Comment by anthonyma94** [comment]
Sorry about that, I'll include some examples:

This is valid since there is text in the description
![image](https://user-images.githubusercontent.com/52266992/129395094-a5ac5868-8580-43a1-8a37-f127b4c6111e.png)

This is invalid since there is no text in the description
![image](https://user-images.githubusercontent.com/52266992/129395164-6aa6d4b5-bf01-40c8-a209-f7080440698e.png)

Currently if the second PR is submitted and the `excludeTitle` flag is `true`, the action succeeds with the message `No commits found in the payload, skipping check.`

**Comment by gilbertsoft** [comment]
If you like to enforce a description something like this may do it:

"$.+\n\n.+"

Does this help or do you still need something else?

**Comment by anthonyma94** [comment]
Unfortunately that doesn't work. If the description is empty while the `excludeTitle` flag is set to `true`, the action skips any checks and displays `No commits found in the payload, skipping check.`

**Comment by gilbertsoft** [comment]
> 
> 
> Unfortunately that doesn't work. If the description is empty while the `excludeTitle` flag is set to `true`, the action skips any checks and displays `No commits found in the payload, skipping check.`

Why do you need to set `excludeTitle` at all? And did you set excludeTitle to true? Without only the PR descriptioin is takken into account.

![grafik](https://user-images.githubusercontent.com/25326036/129699166-3d8951c1-83a2-4a90-b576-7251105cfd14.png)

**Comment by anthonyma94** [comment]
I wanted to check for description based on the title regex. Do you have any suggestions for that?
